aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/actions/AuthenticationCognitoActions.kt

```diff
@@ -35,7 +36,12 @@ object AuthenticationCognitoActions : AuthenticationActions {
             logger.verbose("$id Starting execution")
             val evt = when (val credentials = event.storedCredentials) {
                 is AmplifyCredential.UserPoolTypeCredential -> {
-                    AuthenticationEvent(AuthenticationEvent.EventType.InitializedSignedIn(credentials.signedInData))
+                    AuthenticationEvent(
+                        AuthenticationEvent.EventType.InitializedSignedIn(
+                            credentials.signedInData,
+                            DeviceMetadata.Empty
```

Comment:
What if device key is present when app is restarted after sign in with device SRP?

Reply:
The deviceKey should be accessed by loadCredentials using AmplifyCredentialType.DEVICE_CREDENTIAL ?

Reply:
Fixed